packages/ens-utils/tsconfig.json

```diff
@@ -2,8 +2,8 @@
   "compilerOptions": {
     "target": "ES2020",
     "lib": ["ES2020", "dom"],
-    "module": "ES2020",
-    "moduleResolution": "node",
+    "module": "ES6",
+    "moduleResolution": "node10",
```

Comment:
Hmm.. Is there a special reason for this? https://www.typescriptlang.org/tsconfig/#moduleResolution

![CleanShot 2024-07-09 at 22 23 50](https://github.com/namehash/nameguard/assets/126201998/63d6da1b-fad6-4150-af6f-b0888ff1b9f0)